### ISSUE TITLE
Bug in examples?

### ISSUE THREAD

**Issue by praveen-palanisamy** [google/jax#31]
The [mnist_classifier_fromscratch example](../blob/master/examples/mnist_classifier_fromscratch.py) has a bug in this line:
https://github.com/google/jax/blob/2df36f7510e5cd3c7a3be639c7827d2ff3ae0c01/examples/mnist_classifier_fromscratch.py#L42

The bug also exists on the main page of the project (README.md).

**Comment by praveen-palanisamy** [comment]
For other's reference:
Bug in examples? No.  Looking at the whole (3 lines :D) context makes sense:
https://github.com/google/jax/blob/2df36f7510e5cd3c7a3be639c7827d2ff3ae0c01/examples/mnist_classifier_fromscratch.py#L39-L43

After the for loop, `outputs` is the logits from the final layer.
